bunq/sdk/client.py

```diff
@@ -13,6 +13,11 @@
 from bunq.sdk import context
 from bunq.sdk import exception
 
+try:
```

Comment:
Why are we trying to import?

Reply:
It was needed to import different libs depending on the python version (one for 2^, the other for 3^). Removed the if-elses!